x-pack/solutions/search/plugins/search_homepage/public/components/ai_search_capabilities/vector_search.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { AISearchWorkflow } from './ai_search_workflow';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+
+export const VectorSearch: React.FC = () => {
+  const assetBasePath = useAssetBasePath();
+
+  const vectorSearchText = {
+    image: `${assetBasePath}/vector_search.svg`,
+    imageAlt: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.vectorSearch.imageAlt', {
+      defaultMessage: 'Vector Search',
+    }),
+    heading: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.vectorSearch.title', {
+      defaultMessage: 'Store and search your vector embeddings',
```

Comment:
>      defaultMessage: 'Store and search your vector embeddings',

IMO this is another scenario where the text is talking about the broad benefits of vector search, but the button only takes you to the first small step. If that's an accurate assessment of the current limited workflow, I think it would be helpful to link to https://www.elastic.co/what-is/vector-search for the hype and  https://www.elastic.co/docs/solutions/search/vector for the overall workflow.